javalib/src/main/scala/java/net/URI.scala

```diff
@@ -924,6 +924,11 @@ final class URI private () extends Comparable[URI] with Serializable {
 
   def isOpaque(): Boolean = opaque
 
+  def toURL(): URL = {
```

Comment:
Is it really needed to implement URL? This would make it impossible to have a third party URL implementation.  I have already confirmed that removing URL works downstream and I would prefer that we match Scala.js.

Reply:
I really view this like `Locale` or `java.time` so that third party implementations can occur if needed. We also have the https://github.com/scala-native/scala-native-java-stubs or we could have something else outside of core if we need it.

Reply:
Thanks!
I think I was misunderstood @WojciechMazur comments in https://github.com/scala-native/scala-native/pull/3348#discussion_r1284162278 😅